Applications/MmrMobile/Contracts/IMmrMobileClient.cs

```diff
@@ -36,25 +36,27 @@ public interface IMmrMobileClient
         /// request for one machine and a (comma separated) list of nodes the last know value
         /// for a given point in time
         /// </summary>
-        /// <param name="machineNumber"></param>
-        /// <param name="node"></param>
-        /// <param name="timestamp"></param>
         /// <returns></returns>
         Task<MmrNodeData> GetPointInTimeValuesFromMachine(string machineNumber, string node, DateTime timestamp);
 
         /// <summary>
         /// Get for a given duration and a given list of nodes all values reported from the machine
         /// </summary>
-        /// <param name="machineNumber"></param>
-        /// <param name="node"></param>
-        /// <param name="from"></param>
-        /// <param name="to"></param>
-        /// <param name="take"></param>
-        /// <param name="skip"></param>
         /// <returns></returns>
         Task<MmrNodeData> GetTimeSeriesFromMachine(string machineNumber, string node, DateTime from, DateTime to, int take = 1000, int skip = 0);
         #endregion
 
+        #region eventdata
+
+        /// <summary>
+        /// get all  events from the requested timespan
+        /// to discuss: we should show also events, that overlap the requested duration and adjust the timestamps to fit into this period.
```

Comment:
I think we should return all events which are in the specified timeframe active. The starttime might be before the specified from time.